### ISSUE TITLE
Will the project continue to be updated?

### ISSUE THREAD

**Issue by songwanlong** [nim4/DBShield#8]
hi，
Will the project continue to be updated?

**Comment by nim4** [comment]
Yes It will, I'm so busy with my day job at the moment, but I will have more free time in a month or so. in the mean time any contribute is appreciated
